README.md

```diff
@@ -1,5 +1,52 @@
 # docassemble.assemblylinewizard
 
+## Developing Tests
+
+To write and run the tests, you'll need to set up your testing environment in Python.
+
+### Set up a virtual environment
+
+First, set up a virtual environment called `docassemble` to hold all the packages
+related to this repository.
+This will keep your code clean, and make sure this repository does not interfere with
+your other projects.
+
+```
+$ cd docassemble/assemblylinewizard
+$ pip3 install virtualenv
+$ virtualenv -p $(which python3) docassemble
+$ source docassemble/bin/activate
+$ pip install -r requirements.txt

```

Comment:
What seems to be working better for me is `pip install -r docassemble/assemblylinewizard/requirements.txt`

Reply:
Wonder if it's worth putting in 'If your errors show something like `mysql_config: command not found` it means you're missing a dependency. You may have to look it up to find how to install it for your system.' And maybe 'Contact us if you have trouble' or something. Downside is we can't really account for every issue they're going to have.

Also, I'm still trying to work out how to install the dependencies I need.

Reply:
For the first comment, did you `cd docassemble/assemblylinewizard` (from line 15)? The instructions assume you do that first, although I can rewrite them to all take place in the root directory if you prefer. 

For the second comment, agreed. `mysql` is also a requirement.

Reply:
I've been having issues with setting up/installing the docassemble for python, but that has nothing to do with the setup instructions here. 
The set up instructions are clear, and I think in terms of stream-lining the docassemble aspect, its possible we'll have to wait to see what kind of bugs occur with this.